tests/integration/API/JSON/v1/FormsTest.php

```diff
@@ -186,40 +192,41 @@ public function testItCanExtractListsFromNestedListSelectionBlock() {
   }
 
   public function testItCanRestoreAForm() {
-    $this->form1->trash();
+    $this->form1->setDeletedAt(new \DateTime());
+    $this->formsRepository->flush();
 
-    $trashedForm = Form::findOne($this->form1->id);
+    $trashedForm = Form::findOne($this->form1->getId());
     assert($trashedForm instanceof Form);
     expect($trashedForm->deletedAt)->notNull();
 
-    $response = $this->endpoint->restore(['id' => $this->form1->id]);
+    $response = $this->endpoint->restore(['id' => $this->form1->getId()]);
     expect($response->status)->equals(APIResponse::STATUS_OK);
     expect($response->data)->equals(
-      $this->reloadForm((int)$this->form1->id)->asArray()
+      $this->reloadForm((int)$this->form1->getId())->asArray()
     );
     expect($response->data['deleted_at'])->null();
     expect($response->meta['count'])->equals(1);
   }
 
   public function testItCanTrashAForm() {
-    $response = $this->endpoint->trash(['id' => $this->form2->id]);
+    $response = $this->endpoint->trash(['id' => $this->form2->getId()]);
     expect($response->status)->equals(APIResponse::STATUS_OK);
     expect($response->data)->equals(
-      $this->reloadForm((int)$this->form2->id)->asArray()
+      $this->reloadForm((int)$this->form2->getId())->asArray()
     );
     expect($response->data['deleted_at'])->notNull();
     expect($response->meta['count'])->equals(1);
   }
 
   public function testItCanDeleteAForm() {
-    $response = $this->endpoint->delete(['id' => $this->form3->id]);
+    $response = $this->endpoint->delete(['id' => $this->form3->getId()]);
     expect($response->data)->isEmpty();
     expect($response->status)->equals(APIResponse::STATUS_OK);
     expect($response->meta['count'])->equals(1);
   }
 
   public function testItCanDuplicateAForm() {
-    $response = $this->endpoint->duplicate(['id' => $this->form1->id]);
+    $response = $this->endpoint->duplicate(['id' => $this->form1->getId()]);
     expect($response->status)->equals(APIResponse::STATUS_OK);
     $form = Form::where('name', 'Copy of Form 1')->findOne();
```

Comment:
Could you please replace also this old model usage? You can use `$this->formsRepository`
`$form = $this->formsRepository->findOneBy(['name' => 'Copy of Form 1']);`